### PR TITLE
Add Northern Ireland

### PR DESCRIPTION
**Pull request by systemcatch**
ENTSOE data for Northern Ireland has been absent for a while.  So I've built a parser that gets data from [SONI](http://www.soni.ltd.uk/InformationCentre/) directly.  Exchanges with GB and IE have been implemented but you should definitely check to make sure everything is going in the right direction.

#610

### CODE REVIEW

File: config/exchanges.json [hunk 1]
```diff
@@ -675,7 +675,8 @@
       54.87826
     ],
     "parsers": {
-      "exchange": "ENTSOE.fetch_exchange"
+      "exchange": "ENTSOE.fetch_exchange",
```
Comment (corradio) [inline config/exchanges.json]: You can't have twice the same key: just remove the ENTSOE one.

File: config/exchanges.json [hunk 2]
```diff
@@ -700,9 +701,15 @@
...
   },
   "GB-NIR->IE": {
+    "lonlat": [
+      -7.4,
+      54.2
+    ],
     "parsers": {
-      "exchange": "ENTSOE.fetch_exchange"
-    }
+      "exchange": "ENTSOE.fetch_exchange",
```
Comment (corradio) [inline config/exchanges.json]: Same here

File: config/zones.json [hunk 3]
```diff
@@ -680,10 +680,12 @@
     "parsers": {
       "consumption": "ENTSOE.fetch_consumption",
       "price": "ENTSOE.fetch_price",
-      "production": "ENTSOE.fetch_production"
+      "production": "ENTSOE.fetch_production",
```
Comment (corradio) [inline config/zones.json]: Same here

File: config/exchanges.json (hunk 1 above)
Comment (systemcatch) [reply config/exchanges.json]: Ok I thought we agreed to leave ENTSOE in for when it starts working again.

Comment (corradio) [reply config/exchanges.json]: Sorry about the misunderstanding. For now the system can't handle multiple parsers. We could do it though.
Maybe we could have an array of parsers here. I'll see if it makes sense.
Let's continue with only your parser for now.

Comment (systemcatch) [reply config/exchanges.json]: Gotcha will do.

**Pull request merged**